Git??? (#48)
lithekod/site-v2: website/pages/git_en.md

> # git Workshop

Reviewer: ```suggestion
# git Workshop

```

It's time to sharpen your CV, acquire competences that
keep on giving and enjoy a pleasant event with LiTHe-kod!
The workshop will teach git in an interactive manner, that is
fit for git-gurus and novices alike.

> The 3rd of February 17:30-19:00 LiTHe-kod will be hosting

Reviewer: ```suggestion
The 3rd of February 5:30pm to 7:00pm LiTHe-kod will be hosting
```

> The 3rd of February 17:30-19:00 LiTHe-kod will be hosting

Reviewer 2: ```suggestion
The 3rd of February 5.30pm to 7.00pm LiTHe-kod will be hosting
```

> a git workshop on Zoom! To get there, simple visit this page on the 3rd of februari at 17:30.

Reviewer: ```suggestion
a git workshop on Zoom! To get there, simply visit this page on the 3rd of february at 5:30pm.
```

Reviewer 3: February med stort F

> a git workshop on Zoom! To get there, simple visit this page on the 3rd of februari at 17:30.

Reviewer 2: ```suggestion
a git workshop on Zoom! To get there, simply visit this page on the 3rd of Februari at 5.30pm.
```

> You will be redirected using the magic of the internets and arrive at stimulating Zoom-meeting.

Reviewer: ```suggestion
You will be redirected using the magic of the internet and arrive at a stimulating Zoom-meeting.
```

> The workshop consists of 2 parts. One theoretical part, a lecture for ~45 minutes where

Reviewer: ```suggestion
The workshop consists of 2 parts. One theoretical part: a lecture for ~45 minutes where
```

commands and theory are taught. The second part is a collection of interactive

> exercises where you can apply the lecture on smaller exercises. We will ofcourse be present

Reviewer: ```suggestion
exercises where you can apply your new skills. We will be present
```

> to answer any question you may have.

Reviewer 3: ```suggestion
to answer any questions you may have.
```

# git Workshop

It's time to sharpen your CV, acquire competences that
keep on giving and enjoy a pleasant event with LiTHe-kod!

> The workshop will teach git in an interactive manner, that is

Reviewer 3: ```suggestion
The workshop will teach git in an interactive manner that is
```

fit for git-gurus and novices alike.

> The 3rd of February 5:30pm to 7:00pm LiTHe-kod will be hosting

Reviewer 3: ```suggestion
The 3rd of February from 5:30pm to 7:00pm LiTHe kod will be hosting
```

a git workshop on Zoom! To get there, simply visit this page on the 3rd of February at 5:30pm.

> You will be redirected using the magic of the internet and arrive at a stimulating Zoom-meeting.

Reviewer 3: ```suggestion
You will be redirected using the magic of the internet and arrive at a stimulating Zoom meeting.
```
